cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -31,6 +44,8 @@ import qualified Cardano.Ledger.SafeHash as SafeHash
 import qualified Cardano.Ledger.Shelley.Constraints as Shelley
 import           Cardano.Slotting.Slot (SlotNo (..))
 import qualified Ouroboros.Consensus.Shelley.Eras as Consensus
+import qualified PlutusCore.Evaluation.Machine.ExBudgeting as Plutus
+import qualified PlutusCore.Evaluation.Machine.ExBudgetingDefaults as Plutus
```

Comment:
Are you sure the types we're using here are not exported via `Plutus.V1.Ledger.Api`?

Reply:
These functions are not exported:

```

src/Cardano/Api/Orphans.hs:356:14: error:
    Not in scope: ‘Plutus.extractModelParams’
    Module ‘Plutus.V1.Ledger.Api’ does not export ‘extractModelParams’.
    |
356 |         case Plutus.extractModelParams Plutus.defaultCostModel of
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^

src/Cardano/Api/Orphans.hs:356:40: error:
    Not in scope: ‘Plutus.defaultCostModel’
    Perhaps you meant ‘Plutus.defaultCostModelParams’ (imported from Plutus.V1.Ledger.Api)
    Module ‘Plutus.V1.Ledger.Api’ does not export ‘defaultCostModel’.
    |
356 |         case Plutus.extractModelParams Plutus.defaultCostModel of
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I created a `TODO` for this item so it can be done as a separate PR.

Reply:
I think you should be able to use `defaultCekCostModelParams` here, which is exported from `Plutus.V1.Ledger.Api`.  See https://input-output-rnd.slack.com/archives/C21UF2WVC/p1621890076035500 . We can rename that to something that makes more sense from the viewpoint of the ledger if that would help.

Reply:
Fixed!